config/env/development.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
   //   connection: 'someMongodbServer'
   // }
   blueprints: {
-    rest: true,
+    rest: false,
+    shortcuts: false,
```

Comment:
This is sort of annoying but the catch all wreaks havoc on the api calls from blueprints, overriding all of them. I'm sure there's got to be a better way of doing this (converting to post instead of get, which the http requests use), but this works for now.

All previous logic has moved to test controller functions here, https://github.com/cuttle-cards/cuttle/pull/891/files#diff-ff18ca6c9bd29eacdb89029e8f309522bd67e49d21b18e18f42de044a48c20e4R58-R78.